adagrams/game.py

```diff
@@ -1,13 +1,76 @@
+import random
 from random import randint
 
 def draw_letters():
-    pass
+    letter = []
+    pool = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", 
+        "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    pool_dic= {"A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, 
+    "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, 
+    "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1}
```

Comment:
A few things here: 
- Operators like `=` should be surrounded with white spaces. Take a look at the [PEP8 style guide on white spaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).  In industry, code that doesn't follow an agreed upon code styles will need to be revised to ensure the codebase is neat and consistent.
- To break up long lines of code (`line 8`), we should use multiple lines (see [here](https://peps.python.org/pep-0008/#code-lay-out) in the style guide about using multilines and indentation to create data structures).
- Given that `pool_dic` is a variable that won't be changed, we should follow the python convention of naming it in all uppercase to signal to other developers that this variable should not be modified or re-assigned. 
```suggestion
    POOL_DIC = {
    "A":9,"B":2,"C":2,"D":4,
    "E":12,"F":2,"G":3,"H":2,
    "I":9,"J":1,"K":1,"L":4,"M":2,
    "N":6,"O":8,"P":2,"Q":1,
    "R":6,"S":4,"T":6,"U":4,
    "V":2,"W":2,"X":1,
    "Y":2,"Z":1
}
```